com.unity.sg2/Editor/GraphDeltaRegistry/FunctionDefinitions/StandardDefinitions/Utility/HDRP/Water/BlendWaterNormal.cs

```diff
@@ -0,0 +1,49 @@
+using Usage = UnityEditor.ShaderGraph.GraphDelta.GraphType.Usage;
+
+namespace UnityEditor.ShaderGraph.Defs
+{
+    internal class BlendWaterNormalNode : IStandardNode
+    {
+        public static string Name => "BlendWaterNormal";
+        public static int Version => 1;
+
+        public static FunctionDescriptor FunctionDescriptor => new(
+            Name,
+            @"   NormalWS = BlendWaterNormal(NormalTS, NormalWSin);",
+            new ParameterDescriptor[]
+            {
+                new ParameterDescriptor("NormalTS", TYPE.Vec3, Usage.In),
```

Comment:
NormalTS and NormalWS inputs have float3(1,1,1) as defualt inputs in SG1

Reply:
Good one!  Fixed it.